packages/flame/lib/src/components/scroll_text_box_component.dart

```diff
@@ -39,6 +40,7 @@ class ScrollTextBoxComponent<T extends TextRenderer> extends PositionComponent {
     super.priority,
     super.key,
     List<Component>? children,
+    void Function()? onFinished,
```

Comment:
To be consistent with the rest of these types of functions in the Flame codebase we should probably call it `onComplete` instead.